README.md

```diff
@@ -177,7 +177,7 @@ $ docker run -it --rm go-python/gopy
 - wrap `go` structs into `python` classes **[DONE]**
 - better pythonization: turn `go` `errors` into `python` exceptions **[DONE]**
 - wrap arrays and slices into types implementing `tp_as_sequence` **[DONE]**
-- only `python-2` supported for now
+- only `python-2` supported for now **[DONE]**
```

Comment:
that's true (it works on my machine (TM)), but a bit premature.
we probably want to wait for go-python/gopy#134